generator/src/main/java/org/corfudb/generator/distributions/OperationCount.java

```diff
@@ -0,0 +1,31 @@
+package org.corfudb.generator.distributions;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * This class implements the distribution of a random integer generator.
+ *
+ * Created by maithem on 7/14/17.
+ */
+public class OperationCount extends DataSet {
+    private Random rand = new Random();
+
+    public void populate() {
+        //no-op
+    }
+
+    public List<Integer> sample(int num) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)